recipes/dealii/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "dealii" %}
+{% set version = "9.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://dealii.kyomu.43-1.org/downloads/dealii-{{ version }}.tar.gz
+  sha256: df2f0d666f2224be07e3741c0e8e02132fd67ea4579cd16a2429f7416146ee64
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    # if your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+  host:
+  #run:
+  #  - python
+
+test:
+  requires:
+    - cmake
+    - make
+  source_files:
+    - '*'
+  commands:
+    - 'cd build && make test'
```

Comment:
We usually run this in `build.sh` and test that libraries and include files were installed here. (eg: https://github.com/conda-forge/symengine-feedstock/blob/master/recipe/meta.yaml#L42)

Reply:
Thanks for the clarification. I am thinking about the same thing, since caching `CMake` build directory did not work in the first place.

cf. https://cmake.org/pipermail/cmake/2012-May/050139.html